backend/test/test_org_subs.py

```diff
@@ -206,6 +206,9 @@ def test_login_existing_user_for_invite():
     assert data["firstOrgAdmin"] == True
     assert data["orgName"] == data["oid"]
     assert data["orgName"] == data["orgSlug"]
+    assert data["fromSuperuser"] == True
```

Comment:
Maybe we should add a test where this is false as well? (i.e. have the org admin user invite a user and check the response?)

Reply:
Added additional tests in: https://github.com/webrecorder/browsertrix/pull/2087/commits/0ccbd4807a8feff6035a3bbe233b270effe260ae